server.go

```diff
@@ -120,6 +123,8 @@ func NewFromConfig(conf Config) (ret Server, err error) {
 	return
 }
 
+// HandlePacket processes each packet that is sent to the server, and sends to an
+// appropriate worker (EventWorker or Worker).
```

Comment:
Nit: Might be worth nothing that this is very performance sensitive since in some cases we may process a lot of packets and slowdowns here will lead to drops!